Makefile

```diff
@@ -102,12 +102,12 @@ man: $(MAN8)
 	for a in $^; do gzip -f -9 -c $$a > $$a.gz; done
 
 install: $(PROGS) man
-	mkdir -p $(SBINDIR)
-	mkdir -p $(BINDIR)
-	install -m 750 mced $(SBINDIR)
-	install -m 755 mce_listen $(BINDIR)
-	mkdir -p $(MAN8DIR)
-	install -m 644 $(MAN8GZ) $(MAN8DIR)
+	mkdir -p $(DESTDIR)$(SBINDIR)
```

Comment:
I am confused about this.  Now the user has 2 inputs?  PREFIX and DESTDIR?  Why not set PREFIX conditionally above, and just have the one?

e.g.

make # -> /usr/local/...
make PREFIX=/opt # -> /opt/...

Reply:
Agreed.  Sorry I was blindly applying internal patches.